Add plugin: Global Proxy

windingblack opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin:

## Release Checklist
- [ ] I have tested the plugin on
  - [x]  Windows
  - [ ]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [ ] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [ ] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

joethei commented:
["description": "Make Obsidian supports network proxy",](https://github.com/windingblack/obsidian-global-proxy/blob/664b8a85790835a71b0766fac2bd7c22be509d5e/manifest.json#L6)
Obsidian already supports network proxies, it uses whatever proxy is configured in the settings of the operating system.
Please update your description.

[THIS IS A GENERATED/BUNDLED FILE BY ESBUILD](https://github.com/windingblack/obsidian-global-proxy/blob/664b8a85790835a71b0766fac2bd7c22be509d5e/main.js#L1-L29)
Did you use TypeScript or JavaScript to build your plugin?
If you used Typescript, please add the sourcecode to the repo and remove the `main.js`, if you used JavaScript, please cleanup the code so it does not contain the selected code.

windingblack commented:
Hello, I followed your instructions and cleaned up the specified useless code, and changed the description in `community-plugin.json`, please review it again
